src/test/java/uk/gov/pay/directdebit/gatewayaccounts/resources/GatewayAccountResourceIT.java

```diff
@@ -89,6 +92,32 @@ public void shouldReturnAGatewayAccountIfItExists() {
                 .body(TYPE_KEY, is(TYPE.toString()))
                 .body(EXTERNAL_ID_KEY, is(EXTERNAL_ID))
                 .body(DESCRIPTION_KEY, is(DESCRIPTION))
+                .body(IS_LINKED, is(true))
+                .body(ANALYTICS_ID_KEY, is(ANALYTICS_ID));
+    }
+    
+    @Test
+    public void isLinkedShouldBeFalseIfGatewayAccountHasNoAccessToken() {
```

Comment:
Should there also be a test that checks it’s linked if there is an access token?

Reply:
i added a test a few lines above (see line 95)

Reply:
Ah, missed that, sorry. It wasn’t too noticeable from the diff.